en/troubleshoot.md

```diff
@@ -104,7 +104,7 @@ When you find some network connection issues between Pods from the log or monito
     tkctl debug -n ${namespace} ${pod_name}
     ```
 
-    After the remote shell is started, use the `dig` command to diagnose the DNS resolution. If the DNS resolution is abnormal, refer to [Debugging DNS Resolution](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/) for troubleshooting.
+    After the remote shell is started, use the `dig` command to diagnose the DNS resolution. If the DNS resolution is abnormal, refer to [Debugging DNS Resolution](https://v1-17.docs.kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/) for troubleshooting.
```

Comment:
I'm not sure if it's okay to put a link of **v1.17** (outdated) here, because most of the docs use K8s v1.18. @DanielZhangQD 
The [latest Chinese page](https://kubernetes.io/zh/docs/tasks/administer-cluster/dns-debugging-resolution/) is still working, so the 404 error in English doc might be just temporary. 

I already filed [an issue](https://github.com/kubernetes/website/issues/21930) in kubernetes/website repo. Let's see if they'll fix this link. We can merge this PR and change the link again when the link's fixed. @CaitinChen What do you think?

Reply:
`https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/` this is also accessible.

Reply:
@DanielZhangQD OK. Let's use this link. Thanks.